ignite/templates/module/create/configs.go

```diff
@@ -32,7 +33,13 @@ func configsProtoModify(opts ConfigsOptions) genny.RunFn {
 		if err != nil {
 			return errors.Errorf("couldn't find message 'Module' in %s: %w", path, err)
 		}
+
 		for _, paramField := range opts.Configs {
+			_, err := protoutil.GetFieldByName(params, paramField.Name.LowerCamel)
+			if err == nil {
```

Comment:
is this right?

```suggestion
			if err != nil {
```

Reply:
Yea, if err is nil then it is found, which means duplicated.